### DISCUSSION TITLE
2023.11.21 第十五讲 随意发散

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#38] [Announcements]
https://www.clubhouse.com/room/xn7gVqoX?utm_medium=ch_room_pt&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-991844

**Comment by Lewisrxliu** [comment]
昨晚听这个新国大EMBA讲座的录音，只听了一半， 傅强【长期利率的迷思与全球化的悖论】 https://wplpd.h5.xeknow.com/sl/9z0Su Facebook：https://www.facebook.com/events/856337199310426/
总结
https://mp.weixin.qq.com/s/IypgH7qQvZn2J5ZJD8HWbQ

**Comment by guttasingapore** [comment]
关键词与人的主动引导 成为AI应用的重要出发点，影响未来的路径需要思考， XP的一阶向二阶，如何求导？ 需要函数和建模，建议年轻人进入能够产生正反馈的环境中，Sam Altman in YC 成长之路，
感谢hengsir的推荐，我需要好好读书，准备好与年轻人的对话
Hackers and Painters: Big Ideas from the Computer Age, Paul Graham
Political theory, Paul Graham
Introduction to Political Ideologies, Paul Graham
Well, What Is Photography: A Lecture on Photography on the Occasion of the 10th Anniversary of Fotomuseum Winterthur, Paul Graham
Films, Paul Graham
Free Culture: How Big Media Uses Technology and the Law to Lock Down Culture and Control Creativity
Book by Lawrence Lessig

**Reply by guttasingapore** [reply]
感谢旋转门的hengsir, 
致敬艾萨克·阿西莫夫Isaac Asimov
致敬Paul Graham的书单
https://www.readthistwice.com/person/paul-graham
致敬经典2001-A Space Odyssey （1968）

**Reply by guttasingapore** [reply]
我将无我，不负人民
https://zhuanlan.zhihu.com/p/367769360
http://www.dangjian.cn/shouye/dangjianyaowen/202303/t20230303_6568883.shtml
https://www.bbc.com/zhongwen/simp/chinese-news-47782187

**Reply by guttasingapore** [reply]
相见欢，举杯到12点? 线性或非线性? 阿根廷为例， 国家暴力与金融秩序的重构
thanks xu's input 让我们拭目以待吧
https://www.bbc.com/zhongwen/simp/world-67471770
https://www.youtube.com/watch?v=N8znnztfMEU

**Reply by guttasingapore** [reply]
当今世界上所有国家的四象限：发达国家，发展中国家，日本，阿根廷，
关注日本， thanks xu's input：大事件日元要升息，最近一个月内发生， 让我们拭目以待吧

**Reply by Lewisrxliu** [reply]
> 我将无我，不负人民
> 
> https://zhuanlan.zhihu.com/p/367769360
> 
> http://www.dangjian.cn/shouye/dangjianyaowen/202303/t20230303_6568883.shtml
> 
> https://www.bbc.com/zhongwen/simp/chinese-news-47782187

 对标 “知我罪我，其惟春秋”， 感谢阿叔解释

**Comment by Lewisrxliu** [comment]
关于焦虑： 焦虑是普遍现象， 焦虑会产生巨大需求.....

**Comment by Lewisrxliu** [comment]
我的学习计划之今天：会计学基础，囫囵吞枣看完一本简体中文职高教材版

**Comment by Lewisrxliu** [comment]
继续随意发散：生活不止眼前的苟且，所有人都期望诗和远方，我觉得还要加上咖啡美酒和美食 美女帅哥 旅行 朋友 幽默和快乐
